libs/transloco-keys-manager/README.md

```
@@ -0,0 +1,36 @@
+
+Extract translatable keys from projects that use Transloco.
+
+> This library is being migrated from [jsverse/transloco-keys-manager](https://github.com/jsverse/transloco-keys-manager) into this monorepo.
+
+Discover more about this tool in the [official Transloco documentation](https://jsverse.gitbook.io/transloco/tools/keys-manager-tkm).
+
+> [!IMPORTANT]
+> The Transloco packages are now published under the **@jsverse** scope, update your dependencies to get the latest features 🚀
+
+<p align="center">
+ <img width="50%" height="50%" src="./logo.png">
```

markdownlint-cli2 (0.23.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)